data/src/main/java/com/teamdontbe/data/dto/response/ResponseNotificationListDto.kt

```diff
@@ -0,0 +1,40 @@
+package com.teamdontbe.data.dto.response
+
+import com.teamdontbe.domain.entity.NotiEntity
+import kotlinx.serialization.SerialName
+import kotlinx.serialization.Serializable
+
+@Serializable
+data class ResponseNotificationListDto(
+    @SerialName("memberId")
+    val memberId: Int,
+    @SerialName("memberNickname")
+    val memberNickname: String,
+    @SerialName("triggerMemberNickname")
+    val triggerMemberNickname: String,
+    @SerialName("triggerMemberProfileUrl")
+    val triggerMemberProfileUrl: String,
+    @SerialName("notificationTriggerType")
+    val notificationTriggerType: String,
+    @SerialName("time")
+    val time: String,
+    @SerialName("notificationTriggerId")
+    val notificationTriggerId: Int,
+    @SerialName("notificationText")
+    val notificationText: String,
+    @SerialName("isNotificationChecked")
+    val isNotificationChecked: Boolean,
+) {
+    fun toNotificationListEntity() =
```

Comment:
```suggestion
    fun toNotificationEntity() =
```
p2 : 사소한 네이밍 -> list가 아니니까

Reply:
수정 완!